src/features/DocumentFormatter.ts

```diff
@@ -139,6 +139,7 @@ class PSDocumentFormattingEditProvider implements DocumentFormattingEditProvider
     private readonly ruleOrder: string[] = [
         "PSPlaceCloseBrace",
         "PSPlaceOpenBrace",
+        "PSUseWhitespace",
```

Comment:
I think consistency would be nice between the two names.  "PSUseIndentation/Whitespace" doesn't sound specific enough to me; the "Consistent" designation helps with that.  I've tried thinking of other names and I can't come up with anything better at the moment.